### ISSUE TITLE
Feature request: traffic information

### ISSUE THREAD

**Issue by MBj1703** [osmandapp/OsmAnd-iOS#3975]
The system snt me yesterday directly into a full closure on the highway.
Would be nice to have traffic information and with it some rerouting.

**Comment by vinolent** [comment]
Agree. Traffic information and the ability to automatically reroute based on traffic conditions is pretty essential and expected in a sat nav application.

**Comment by yuriiurshuliak** [comment]
Possibly duplicate of #https://github.com/osmandapp/OsmAnd/issues/6878
